src/loginPage/LoginPage.js

```diff
@@ -1,15 +1,65 @@
-import React from 'react';
+import React, {useState} from 'react';
+import {useNavigate} from 'react-router-dom';
+
+//TODO: replace to a separate file
+async function newSession(login, password){
+    //TODO: replace to constants
+    const SERVER_PATH = 'http://localhost:5000';
+    const endPoint = SERVER_PATH + '/sessions/login';
+
+    return await (fetch(
+        endPoint,
+        { method: 'post',
+            body: JSON.stringify({ login, password }),
+            headers: { 'Content-type': 'application/json'}
+        })
+        .then(res => {
+            if (res.ok)
+                return res.json();
+            throw res;
+        }).then(data => {
+            return { status: data.status, sessionID: data.sessionID, userRole: data.role};
```

Comment:
Try to avoid unnecessary use of a `return` keyword. To return an object, you may use the following syntax:
`data => ({ ... })`

You may apply it to other changes in this PR